### PR TITLE
Additional content fixes

### PR DESCRIPTION
**Pull request by jaffrepaul**
Assorted content fixes and removal of unused packages.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,6 @@
   },
   "dependencies": {
     "@docusaurus/core": "^2.4.0",
-    "@docusaurus/plugin-google-tag-manager": "^2.4.0",
```
Comment (jaffrepaul) [inline package.json:29]: This is already in the theme and doesn't need to be added as another dep.
https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-google-gtag#installation

File: docs/guides/continuous-integration/gitlab-ci.mdx [hunk 2]
```diff
@@ -276,6 +276,5 @@ A complete CI workflow against multiple browsers, viewports and operating
 systems is available in the Cypress Real World App (RWA).
 
 Clone the <Icon name="github" inline="true" contentType="rwa" /> and refer to
-the
-[.gitlab-ci.yml](https://github.com/cypress-io/cypress-realworld-app/blob/develop/.gitlab-ci.yml)
+the [.gitlab-ci.yml](https://github.com/cypress-io/cypress-realworld-app/blob/develop/.gitlab-ci.yml)
```
Comment (MikeMcC399) [inline docs/guides/continuous-integration/gitlab-ci.mdx:279]: The GitLab example has been deactivated. See also PR https://github.com/cypress-io/cypress-realworld-app/pull/1365.

File: docs/guides/cloud/smart-orchestration/parallelization.mdx [hunk 3]
```diff
@@ -223,7 +223,7 @@ runs can be utilized independently of Cypress parallelization.
 
 :::info
 
-## Cross Browser Testing
+<strong>Cross Browser Testing</strong>
```
Comment (elylucas) [inline docs/guides/cloud/smart-orchestration/parallelization.mdx:226]: was there a reason to using html here to make it bold instead of markdown?

**Review by elylucas — APPROVED** [review]
Asked a curiosity question but lgtm

File: docs/guides/cloud/smart-orchestration/parallelization.mdx (hunk 3 above)
Comment (jaffrepaul) [reply docs/guides/cloud/smart-orchestration/parallelization.mdx:226]: Yeah it a little hacky but using the markdown syntax adds in these admonitions throws additional line height/padding below the title and makes the overall height taller. I think some some of these were done with html in the old site and when we ported over this became clear so we just went with this pattern to keep things more snug until the redesign is accounted for.

File: docs/guides/continuous-integration/gitlab-ci.mdx (hunk 2 above)
Comment (jaffrepaul) [reply docs/guides/continuous-integration/gitlab-ci.mdx:279]: Hadn't seen that. I'll remove this ref. Thanks @MikeMcC399

**Review by MikeMcC399 — APPROVED** [review]
@jaffrepaul 

Thanks for removing the section about the disable GitLab CI in RWA, That looks fine now! 👍🏻

**Pull request merged**